lnwallet/channel.go

```diff
@@ -1893,6 +1893,33 @@ func (lc *LightningChannel) restorePendingRemoteUpdates(
 				payDesc.LogIndex)
 		}
 
+		var height uint64
+		var heightSet bool
+		if payDesc.EntryType != Add {
```

Comment:
Can instead flip this to avoid indentation: 
```go
if payDesc.EntryType == Add {
    continue
}
```

Reply:
Agree